lib/collection/property.js

```diff
@@ -46,9 +47,9 @@ _.inherit((
 
         // first we extract id from all possible sources
         // we also check if this property is marked to require an ID, we generate one if not found.
-        id = (src && src.id) || this.id || (this._ && this._.postman_id) || (this._postman_requiresId && uuid.v4());
+        // eslint-disable-next-line dot-notation
```

Comment:
This change is in accordance with the convention of accessing private intended properties via the `['']` notation only.